src/runtime/runtime_atsamd21g18.go

```diff
@@ -24,6 +24,7 @@ func init() {
 	initRTC()
 	initUARTClock()
 	initI2CClock()
+	initPWMClocks()
```

Comment:
Perhaps this could be moved into `InitPWM`? I suspect not initializing the clocks will reduce power consumption.